DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/ReferencePointConverter.cs

```diff
@@ -55,10 +56,15 @@ public DB.Transform GetReferencePointTransform(string referencePointSetting)
   {
     // first get the main doc base points and reference setting transform
     var referencePointTransform = DB.Transform.Identity;
+
+    // POC: bogus disposal below
+#pragma warning disable CA2000
     var points = new DB.FilteredElementCollector(_contextStack.Current.Document.Document)
       .OfClass(typeof(DB.BasePoint))
       .Cast<DB.BasePoint>()
       .ToList();
+#pragma warning restore CA2000
+
```

Comment:
I wish there was a way to do this better than with #pragmas but it will do

Reply:
Typically, we would split out the collector so we can dispose it.
```
using var collector = new DB.FilteredElementCollector(_contextStack.Current.Document.Document);
var points = collector.OfClass(typeof(BasePoint)).ToList<DB.BasePoint>();
```